ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -53,7 +53,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
+    workflow: cucushift-installer-rehearse-aws-ipi-byo-route53-registry-s3-cloudfront
```

Comment:
Is this change made on purpose?

Reply:
That is not a big deal, it is just a rehearse job only for testing purpose, not our team's regular ci job.